### ISSUE TITLE
DCPR requests must be searchable and become metadata records

### ISSUE THREAD

**Issue by gubuntu** [kartoza/ckanext-dalrrd-emc-dcpr#218]
Although there was a request that as soon as a DCPR request has been approved by the NSIF (i.e. saved / submitted to CSI) it must appear in general searches, i.e. it must become and behave like a public metadata (EMC) record, the regulations (SDI Act Regulations Section 5.1.b) state that this should happen only after the CSI decision. 

It will still need to show its DCPR status, which will update when the CSI approves or rejects it. 

The regulation states "a request...must be registered in the DCPR and also included as provisional metadata in the EMC within ten working days [...of approval or rejection by the CSI]", so the workflow could be that upon CSI approval (or rejection), the record owner (applicant) is notified to log in and complete any outstanding fields in the record. 

Conceptually, a DCPR request is a metadata record with a future date. 

References:
- RF05
- Section 2.2 (DCPR workflows) of requirements specification and subsequent amendments. Note 'Moderated DCPR records shall become part of the EMC and be browsable as public records'

**Comment by Mohab25** [comment]
@gubuntu currently under review #248

**Comment by gubuntu** [comment]
@Samweli @Mohab25 I've tweaked the OP to hopefully accurately reflect better what is required. Please expedite mockups as discussed and think about the best approach to achieve this with minimal refactoring

**Comment by Mohab25** [comment]
based on Nonetobeko's sprint planning meeting and @Samweli feedback, he already done an extensive research on how to integrate EMC and DCPR, this goes beyond showing DCPR requests on EMC page, the search engine should be unified for both systems, and the EMC should have it's own section.

**Comment by Mohab25** [comment]
based on @vutomim  email and @ntobe-nyuswa sprint planning meeting today (Aug 12) and quick brief from@Samweli, he already done an extensive research on how to integrate EMC datasets and DCPR Requests in one search engine, this goes beyond showing DCPR requests on EMC page, the search engine should be unified for both systems, and the EMC should have it's own section.

**Comment by gubuntu** [comment]
how do you mean the EMC should have its own section? We need mockups or diagrams or both to clearly show what we plan and to modify these till there is consensus, before implementing

**Comment by gubuntu** [comment]
to be clear:
<img width="549" alt="Screenshot 2022-08-08 at 11 32 55" src="https://user-images.githubusercontent.com/496911/183387168-e3159e83-b6ac-41e8-a481-d738b5fcf6b3.png">

and such new metadata records that originated via DCPR to appear alongside other metadata records much like this: 
<img width="1172" alt="Screenshot 2022-08-08 at 11 33 43" src="https://user-images.githubusercontent.com/496911/183387365-80ad3545-338c-4f24-8db5-84b1b42cf28d.png">
